controllers/apps/docker/useDocker.js

```diff
@@ -11,33 +11,26 @@ const useDocker = async (apps) => {
     dockerHost: host,
   } = await loadConfig();
 
-  let containers = null;
+  let containers = []
+  const hosts = host.split(';');
 
-  // Get list of containers
   try {
-    if (host.includes('localhost')) {
-      // Use default host
-      let { data } = await axios.get(
+    // Get list of containers
+    containers = (await Promise.all(hosts.reduce((acc, host) => {
+      const isLocalhost = host.includes('localhost');
+      acc.push(axios.get(
         `http://${host}/containers/json?{"status":["running"]}`,
-        {
+        isLocalhost ? {
           socketPath: '/var/run/docker.sock',
-        }
-      );
-
-      containers = data;
-    } else {
-      // Use custom host
-      let { data } = await axios.get(
-        `http://${host}/containers/json?{"status":["running"]}`
-      );
-
-      containers = data;
-    }
-  } catch {
-    logger.log(`Can't connect to the Docker API on ${host}`, 'ERROR');
+        } : {}
+      ))
+      return acc;
+    }, []))).flatMap(({ data }) => data)
+  } catch (err) {
+    logger.log(`Can't connect to the Docker API on one of the hosts: ${hosts.join(', ')}`, 'ERROR');
```

Comment:
I think it's possible to improve this by using `Promise.allSettled` and a custom rejection so at least the hosts that do resolve properly will work - I didn't have time to experiment with this the other night but will have a go at it soon.